hermes-guide-days-website/src/app/footer/footer.component.css

```diff
@@ -0,0 +1,134 @@
+* {
+  color: #fff;
+  font-family: "Montserrat", sans-serif;
+}
+
+.footer {
+  background-color: #333;
```

Comment:
Colors should be added as variables so we can always change the color scheme easily and keep consistency, refer to https://www.youtube.com/watch?v=BfF8evabQus for an example